spec/integration/hanami/controller/sessions_spec.rb

```diff
@@ -58,7 +58,7 @@ def response
   include Rack::Test::Methods
 
   let(:app) do
-    Hanami::Action::Configuration.new
+    # Hanami::Action::Configuration.new
```

Comment:
Is this needed here?

Reply:
Oh yeah, I can clean this up! Thanks for the note :)

(The fact that we no longer need to do this in 2.0 is excellent)